js/src/ui/CopyToClipboard/CopyToClipboard.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+import { IconButton } from 'material-ui';
+import Clipboard from 'react-copy-to-clipboard';
+import CopyIcon from 'material-ui/svg-icons/content/content-copy';
+import Theme from '../Theme';
+const { textColor, disabledTextColor } = Theme.flatButton;
+
+export default class CopyToClipboard extends Component {
```

Comment:
General comment - would prefer the filename as "copyToClipboard.js" since that is actually the standard through the codebase. (Some differently named files have slipped-in along the way, obviously none of us was paying attention).